processor/elasticinframetricsprocessor/processor.go

```diff
@@ -28,6 +28,8 @@ import (
 	"go.uber.org/zap"
 )
 
+const OTelRemappedLabel = "otel_remapped"
```

Comment:
It's the same constant declared in `remappers/common/const.go`, right? I'm wondering if we could we use that one here instead of declaring it again?

Reply:
Could find a way tbh. Nice idea.

On the other hand, if someone wants to test this library is easier to find the value and not check the remote package

Reply:
I noticed that we are already importing the `remappers` package, that's why I asked if we could use that instead.

Reply:
Done in https://github.com/elastic/opentelemetry-collector-components/pull/147/files#diff-d8a33fa92f13418d93d8e208450066b53fe923e4026cc3d184eb3f89bc8d7c62R32